registrations/models.py

```diff
@@ -68,6 +68,21 @@
     (VatPercentage.VAT_10.value, "10 %"),
     (VatPercentage.VAT_0.value, "0 %"),
 )
+VAT_CODE_MAPPING = {
+    VatPercentage.VAT_24.value: "44",
+    VatPercentage.VAT_14.value: "45",
+    VatPercentage.VAT_10.value: "46",
+    VatPercentage.VAT_0.value: "4U",
+}
+
+
+def get_vat_code_mapping_choices():
+    choices = []
+
+    for vat_percentage, vat_label in VAT_PERCENTAGES:
+        choices.append((VAT_CODE_MAPPING[vat_percentage], vat_label))
```

Comment:
```suggestion
    choices = [(VAT_CODE_MAPPING[percentage], label) for percentage, label in VAT_PERCENTAGES]
```
IMO a list comprehension is much cleaner for this kind of stuff. ofc feel free to disagree 🙂

Reply:
Oh yes, sometimes one just forgets to use the more handy stuff. Changed as suggested.